### PR TITLE
Add temporary 2021 speaker info page

### PR DESCRIPTION
**Pull request by tylerdave**
Add a temporary copy of the speaker info page so we can provide a working link to confirmed speakers.

The new version of the full site will be available Soon and this page will exist at the same URL.

### CODE REVIEW

**Comment by tylerdave** [comment]
@mpirnat let me know if you want me to change/add anything or feel free to push to this PR branch

You can preview the speaker info page here: https://deploy-preview-88--pyohio.netlify.app/2021/present/speaker-info/

File: netlify-static/2021/present/speaker-info/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+<ul>
+<li><p><strong>Lightning Talks</strong> should be between <strong>4 - 5 minutes</strong> in length.</p></li>
+<li><p><strong>Thunder Talks</strong> should be between <strong>9 - 10 minutes</strong> in length.</p></li>
+</ul>
+<p>Please, please, please do not go over the maximum time for your talk! If you go over time, we cannot guarantee that the entire talk will be presented on the stream.</p>
+<h3 id="what-format-should-my-video-be">What format should my video be?</h3>
+<ul>
+<li>Format: H264 MP4</li>
+<li>Resolution: 1920x1080</li>
+<li>Frame rate: 30fps</li>
```
Comment (mpirnat) [inline netlify-static/2021/present/speaker-info/index.html:37]:
```suggestion
<li>Format: H264 MP4</li>
<li>Resolution: 1920x1080</li>
<li>Frame rate: 30fps</li>
<li>Audio: AAC (preferred) or MP3</li>
```

File: netlify-static/2021/present/speaker-info/index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          <h1>PyOhio 2021 Speaker Information</h1>
+         
+          <h2 id="video-recording-guidelines">Video Recording Guidelines</h2>
+<p>Congratulations again on having your talk accepted for PyOhio 2021! We’re truly thankful to have you as part of this unusual year of our conference.</p>
+<p>This year, we are asking all speakers to deliver a prerecorded video of their talks. This doc will walk you through the essentials.</p>
+<h3 id="how-long-should-my-video-be">How long should my video be?</h3>
+<p>For PyOhio 2021, we are doing a collection of short talks. To help keep the event on track and running smoothly for all of our remote attendees, we have tighter time requirements than in live events:</p>
+<ul>
+<li><p><strong>Lightning Talks</strong> should be between <strong>4 - 5 minutes</strong> in length.</p></li>
+<li><p><strong>Thunder Talks</strong> should be between <strong>9 - 10 minutes</strong> in length.</p></li>
```
Comment (mpirnat) [inline netlify-static/2021/present/speaker-info/index.html:30]:
```suggestion
<li><p><strong>Lightning Talks</strong> should be <strong>5 minutes ± 30 seconds</strong> in length.</p></li>
<li><p><strong>Thunder Talks</strong> should be <strong>10 minutes ± 30 seconds</strong> in length.</p></li>
```

**Review by mpirnat — APPROVED** [review]
Overall great, thank you for porting this over from last year's site!

I added a couple of suggestions in comments as it's probably the easiest way to add commits to this.

**Review by mpirnat — APPROVED** [review]

**Pull request merged**